sdk/storage/azure-storage-blob-changefeed/azure/storage/blobchangefeed/_deserialize.py

```diff
@@ -0,0 +1,53 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+_SERVICE_PARAMS = {
+    "blob": {"primary": "BlobEndpoint", "secondary": "BlobSecondaryEndpoint"},
+}
+
+
+def parse_connection_str(conn_str, credential, service):
```

Comment:
We already have a dependency on Storage-Blobs being installed - maybe we could just call into that?

Reply:
good catch!